.gitignore

```diff
@@ -1,5 +0,0 @@
-_site
```

Comment:
You shouldn't be removing this file, rather use this to remove folders which you see on your local env that you shouldn't be committing. 

Reply:
What i did was deleted everything in new branch and then recreated from scratch by just copying.
There arent anyfiles that needs to be in .gitignore for this repo. So i dont think we need to add one.